src/lib/update-alternatives/UI/main_dialog.rb

```diff
@@ -46,6 +46,24 @@ def dialog_content
       )
     end
 
+    def help_handler
+      Yast::Popup.LongText(
+        _("Help"),
+        RichText(
+          _("<p>The status of an alternative, can be automatic or manual. " \
+          "In <b>automatic mode</b> the system set the choice with higher priority. " \
```

Comment:
technically correct, but do not help much user. I think it need some explanation who set priority like `Priority is set by packages creators trying to provide reasonable defaults.`